### ISSUE TITLE
Crash with access violation on null pointer

### ISSUE THREAD

**Issue by Jasper-Bekkers** [EpicGames/raddebugger#1]
Thanks for the wonderful tool! I figured I'd try it out immediately, maybe in a somewhat unintended use-case (a random Rust application I had laying around). But after a while of not paying attention the debugger crashed with an access violation on a 0 pointer.

Is there some logging I could give you that would make hunting this down easier?

**Comment by ryanfleury** [comment]
Unfortunately there aren't any logs I can point you to, but the included PDB files should let you attach to the debugger with e.g. Visual Studio, and get a call stack. You can also try the `debug` build included in the `.zip`.

**Comment by Jasper-Bekkers** [comment]
I had built from source on c8723613dfa42ecbd1bd7fee05ebf7088b14e925 fwiw. Unfortunately I didn't have a debugger attached at the time of the crash, nor could I reproduce it later.

I don't think this issue (as reported) will help you much either, exept potentially as a gentle reminder to add some diagnostics support later on.

**Comment by ryanfleury** [comment]
Yeah, makes sense! Will close for now, but if you end up finding a solid repro or a callstack, just re-open and let me know.
